back/boxtribute_server/graph_ql/types.graphql

```diff
@@ -86,6 +86,7 @@ type Product {
   createdOn: Datetime
   lastModifiedBy: User
   lastModifiedOn: Datetime
+  deleted: Datetime
```

Comment:
I modified this file manually. I went through the documentation but I wasn't sure whether this was meant to be manually edited, or whether this is auto-generated. Can you help me with some insights on this? @pylipp

Reply:
Hej @serchtul , this file is part of the definition of our GraphQL API and is manually edited.
By adding this field it's available through the API, you did the correct thing :+1: 
On the FE a TypeScript type definition file is derived by `yarn generate-graphql-ts-types` but I can see you already figured that out. :slightly_smiling_face: